eventually/mentor/views.py

```diff
@@ -62,22 +67,50 @@ def get(self, request):
         available_students = [CustomUser.get_by_id(id).to_dict() for id in available_students]
 
         response = {'my_students': my_students,
-                    'all_students': all_students,
+                    'assigned_students': assigned_students,
                     'available_students': available_students}
 
         return JsonResponse(response, status=200)
 
-
-
     def post(self, request):
         """
         Method that handles POST request.
 
         :param request: the accepted HTTP request.
         :type request: `HttpRequest object`
 
-        :param mentor_id: ID of the certain mentor.
-        :type mentor_id: `int`
+        :return: return JsonResponse within record data with status 201 if record was successfully
+                create
+        """
+
+        data = request.body
+        if not data:
+            return RESPONSE_400_INVALID_DATA
+
+        topic = Topic.get_by_id(data.get('topicId'))
+        if not topic:
+            return RESPONSE_400_INVALID_DATA
+
+        user = request.user
+        if user in topic.mentors.all() or user == topic.author:
+            return RESPONSE_403_ACCESS_DENIED
+        record = list(MentorStudent.objects.filter(student_id=user.id, topic_id=topic.id))
```

Comment:
don't use objects in view

Reply:
added method is_topic_student to mentor models and used it instead of "record = list(MentorStudent.objects.filter(student_id=user.id, topic_id=topic.id))"

Reply:
have rewrite previous changes.
add method topic_student_belonging, that returns MentorStudent instance, if student is assigned to course, or None, if not.
Use this method in Post, Put and is_topic student methods